crates/bevy_ecs/src/world/entity_ref.rs

```diff
@@ -346,6 +415,39 @@ impl<'w> EntityMut<'w> {
         self
     }
 
+    /// Adds a relation to an entity.
+    ///
+    /// This will overwrite previous values of the same relation to the same entity.
+    pub fn add_relation<R: Component>(&mut self, relation: R, target: Entity) -> &mut Self {
```

Comment:
Should this be `insert_relation` for consistency?

Reply:
Sounds good. If you want to merge the current form, I'm fine to leave this as long as you add the matching `remove_relation` method.

Reply:
Added!